pilot/pkg/model/context.go

```diff
@@ -152,9 +153,6 @@ const (
 	// https://github.com/istio/istio/blob/master/install/kubernetes/templates/istio-sidecar-injector-configmap-debug.yaml.tmpl#L27
 	ConfigPathDir = "/etc/istio/proxy"
```

Comment:
Should we also have ConfigDir here ?

Reply:
Good question. In old autotools days the default is $PREFIX/etc. I don't think we should strictly follow this approach. /etc is fine for me. Should we ask others?